api/api/signals.py

```diff
@@ -14,6 +14,7 @@
 from wazuh.core.cluster.utils import running_in_master_node
 from wazuh.core.configuration import update_check_is_enabled
 from wazuh.core.manager import query_update_check_service
+from wazuh.core.indexer import get_indexer_client
```

Comment:
This is not currently being used. If it will be, please add a TODO to have it explicitly identified. If not, we should remove it.